ecephys_spike_sorting/modules/automerging/README.md

```diff
@@ -1,12 +1,39 @@
-Automerging
-==============
-Looks for clusters that likely belong to the same cell, and merges them automatically.
+# Automerging
 
-This is not currently part of our pipeline since switching to Kilosort2, but we're keeping the code around in case others find it useful. For example, it could be helpful for matching units across a series of chronic recordings.
+Searches for clusters that likely belong to the same cell, and merges them automatically.
 
+This module has not been used since switching to Kilosort 2, which has far fewer split units than Kilosort 1. We're keeping the code around in case others find it useful. For example, it could be helpful for matching units across a series of chronic recordings. However, much more complete implementations of this functionality exist elsewhere ([UnitMatch](https://github.com/EnnyvanBeest/UnitMatch), for example).
+
+
+### SpikeInterface implementation
+
+SpikeInterface does not currently include the ability to automatically merge units, but this is under active development. Information that is helpful for making merge decisions, such as waveform similarity and cross-correlograms, can be computed using the `postprocessing` module:
```

Comment:
@jsiegle we have something similar that might be worth mentioning: https://spikeinterface.readthedocs.io/en/latest/modules/curation.html#automatic-curation-tools

In particular: https://spikeinterface.readthedocs.io/en/latest/api.html#spikeinterface.curation.get_potential_auto_merge

The `get_potential_auto_merge()` function basically looks for potential merges mased on template similarity, cross-correlgrams, and improved ISI violation quality